derive_stringset/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "derive_stringset"
```

Comment:
Rename to `miniconf` to mirror the repository name, otherwise cargo will fail to find things named miniconf